scripts/quality/rollup_v2/patches/wrong_import_order.py

```
@@ -67,35 +64,50 @@ def generate(
-        )
+        return None
+    return (import_start, import_end)
 
-    import_lines = [lines[i] for i in range(import_start, import_end) if lines[i].strip()]
-    sorted_imports = sorted(import_lines, key=lambda l: (_classify_import(l), l.strip()))
 
-    # Rebuild the import block with group separators
-    grouped: list[str] = []
+def _build_sorted_block(import_lines: List[str]) -> List[str]:
+    """Sort import lines by group and build a block with group separators."""
+    sorted_imports = sorted(import_lines, key=lambda l: (_classify_import(l), l.strip()))
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)